server/bin/pbench-import-datasets.py

```diff
@@ -33,9 +33,9 @@
 from pbench import BadConfig
 from pbench.common.logger import get_pbench_logger
 from pbench.server import PbenchServerConfig
+from pbench.server.api import init_db
```

Comment:
Again, I think this would be more logically consistent as `from pbench.server.database import init_db` since this command (and others like it) have nothing to do with the api. I get that "throwing it in there" was convenient, but I don't think it's clean design.